test-vectors/schema.json

```diff
@@ -0,0 +1,156 @@
+{
+  "$id": "https://filecoin.io/oni/schemas/test-vector.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "a filecoin VM test vector",
```

Comment:
nit: this will likely cover more than just the VM since these tests validate the application of tipsets and/or complete chains.